src/App/Routes/Modals/RegisterModal/MultiWalletConnection/View.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC, PropsWithChildren } from "react";
+
+import Button, { ButtonProps } from "src/components/Button";
+
+type Props = {
+  theme?: ButtonProps["theme"];
+  size?: ButtonProps["size"];
+  onConnect: () => void;
+};
+
+const ConnectButton: FC<PropsWithChildren<Props>> = ({ children, onConnect, size, theme }) => {
+  return (
+    <Button className="min-w-full" theme={theme} onClick={onConnect} size={size}>
+      {children}
+    </Button>
+  );
+};
```

Comment:
It should return what you return in the `./index.tsx` file (see comment on this file)